tox.ini

```diff
@@ -56,13 +56,12 @@ deps =
 commands =
     flake8 kolibri
 
-##### remove the link checker test because it causes random failures
-#[testenv:docs]
-#changedir=docs/
-#deps =
-#    -r{toxinidir}/requirements/docs.txt
-#commands =
-#    sphinx-build -b linkcheck ./ _build/
+[testenv:docs]
+changedir=docs/
```

Comment:
So we weren't running docs before? Will this still speed up our tests if we add it here?